src/__internal__/form-field/form-field.spec.tsx

```diff
@@ -242,4 +254,22 @@ describe("FormField", () => {
       consoleSpy.mockRestore();
     }
   );
+
+  it.each(["error", "warning", "info"])(
```

Comment:
**comment**: I don't think there's any need for this test to be here in `FormField`. It's testing what `Switch` does or doesn't do with particular prop combinations - that should be in the `Switch` tests (and I think you do already have this there 👍 ).

There's nothing wrong with testing how `FormField` works with your new context (although I don't think it's really needed), but in that case I think rather than rendering `Switch` explicitly we should render `FormField` inside just the context provider. But as I said I don't think this is necessary here.